### ISSUE TITLE
Improve documentation on verifying release artefacts

### ISSUE THREAD

**Issue by theseion** [coreruleset/coreruleset#2435]
I spent far too much time hunting for the public PGP key we use to verify release artefacts (I had to ask @fzipi for help in the end). When he pointed me to the security document, I found the key immediately but there should be clear instructions on how to verify releases.

Verifying releases is something that many (most) developers don't do or don't know how to. I think that it is very important but it is also quite hard. We should make it as simple as possible to verify artefacts and not assume that all users are security experts.

Here's a quick write up for GnuPG:
1. install GnuPG if you don't have it
2. import the CRS PGP key:
    ```bash
    gpg --fetch-key https://coreruleset.org/security.asc
    ```
4. download the artefact together with the signature:
    ```bash
    wget https://github.com/coreruleset/coreruleset/archive/refs/tags/v3.3.2.tar.gz
    wget https://github.com/coreruleset/coreruleset/releases/download/v3.3.2/coreruleset-3.3.2.tar.gz.asc
    ```
5. verify the signature:
    ```bash
    gpg --verify coreruleset-3.3.2.tar.gz.asc v3.3.2.tar.gz
    ```
6. the output should look like this:
    ```bash
    gpg: Signature made Mi 30 Jun 15:05:48 2021 CEST
    gpg:                using RSA key 36006F0E0BA167832158821138EEACA1AB8A6E72
    gpg: Good signature from "OWASP Core Rule Set <security@coreruleset.org>" [unknown]
    gpg: WARNING: This key is not certified with a trusted signature!
    gpg:          There is no indication that the signature belongs to the owner.
    Primary key fingerprint: 3600 6F0E 0BA1 6783 2158  8211 38EE ACA1 AB8A 6E72
    ```

The output above is actually confusing. When you look at it, it seems to say that something is wrong. We need to explain too how to make that go aways or why it's ok.

**Comment by lifeforms** [comment]
Very nice idea, we should definitely put this in our documentation.

The warning is due to the user knowing the key, but not trusting it.
A user can trust the key by:

```sh
gpg edit-key 36006F0E0BA167832158821138EEACA1AB8A6E72
gpg> trust
Your decision: 5
```

**Comment by RedXanadu** [comment]
Added to documentation TODO list (so if no one else gets time to write this up, I will do it when the Developers section is written.)

**Comment by dune73** [comment]
So we could close this issue, actually?

**Comment by RedXanadu** [comment]
Well, it would be amazing if someone could turn the above documentation into a pull request on the documentation repo :) So I'd like to keep this issue open.

Otherwise, eventually I will get to doing this myself while working through the documentation TODO list, and I'll close this issue then if there are no other takers (I can live in hope :smile:)
